app/services/api/v3/participant_declarations_query.rb

```diff
@@ -34,6 +34,11 @@ def participant_declarations
           :participant_profile,
           :cpd_lead_provider,
         )
+        .joins(participant_profile: :induction_records)
```

Comment:
could we add the selection of the mentor profile id to the statement please, also a test :)?